### ISSUE TITLE
JAWS doesn't work with composer at all

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#9746]
I send a message, and then I'm paging up, and then I'm filtering members, then I'm at breadcrumbs.

**Comment by turt2live** [comment]
Upstream issue: https://github.com/ianstormtaylor/slate/issues/2793

**Comment by bwindels** [comment]
OOI, how badly does the edit message composer fare?

**Comment by turt2live** [comment]
I couldn't get that far, so couldn't tell you, sorry

**Comment by t3chguy** [comment]
CIDER has the same issue. After sending a message the caret is still flashing on the composer but the SR goes into some commands mode so further keypresses are intercepted and not treated as input.

**Comment by bwindels** [comment]
do you know what makes the screen reader go into command mode? Is it the Enter from sending, or page up as travis described?

**Comment by t3chguy** [comment]
Enter from sending in my case. Message sent fine, then caret kept blinking and it looked like it was still focused but it was not in compose mode anymore

**Comment by bwindels** [comment]
Looks like it enter "Forms mode" indeed when hitting enter on a field:
https://dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts#jaws-forms

**Comment by t3chguy** [comment]
It might just be how single-line input forms work in JAWS and is desireable to JAWS users

**Comment by bwindels** [comment]
I know a JAWS user, asking him.

**Comment by t3chguy** [comment]
Based on https://github.com/vector-im/riot-web/issues/7119#issuecomment-409627941 that is indeed the expectation, the difference between CIDER and Slate is if in CIDER you hit enter again it'll put you back into forms mode and you can send another message. Reopen if am wrong.
